graylog2-web-interface/src/components/search/MessageFieldExtractorActions.jsx

```diff
@@ -25,17 +25,34 @@ const MessageFieldExtractorActions = React.createClass({
     );
   },
   render() {
-    return (
-      <div className="message-field-actions pull-right">
-        <DropdownButton pullRight
-                     bsSize="xsmall"
-                     title="Select extractor type"
-                     key={1}
-                     id={`select-extractor-type-dropdown-field-${this.props.fieldName}`}>
-          {ExtractorUtils.EXTRACTOR_TYPES.map(extractorType => this._formatExtractorMenuItem(extractorType))}
-        </DropdownButton>
-      </div>
-    );
+    let messageField = this.props.message.fields[this.props.fieldName];
```

Comment:
Small nitpick: can we use `const` instead of `let` in here? The variable is never modified afterwards.